js-src/functions/build.sbt

```diff
@@ -0,0 +1,18 @@
+import org.scalajs.core.tools.linker.backend.OutputMode
+
+enablePlugins(ScalaJSPlugin)
+
+name := "functions"
+
+libraryDependencies ++= Seq(
+  "org.scala-js" %%% "scalajs-dom" % Versions.Dom,
+  "org.querki" %%% "jquery-facade" % Versions.JQuery
+)
+
+// move our output folder to static
+artifactPath in(Compile, fastOptJS) := baseDirectory.value / ".." / ".." / "resources" / "js" / s"scala-${name.value}.js"
+artifactPath in(Compile, fullOptJS) := baseDirectory.value / ".." / ".." / "resources" / "js" / s"scala-${name.value}.js"
```

Comment:
Don't configure `fastOptJS` and `fullOptJS` to produce the same target file. It will confuse caches (`fullOptJS` after `fastOptJS` will not recreate the file, for example).

Reply:
Oh good to know. What's the recommended way to setup a build like this? (Where I'd like my html file to be static and to just import the compiled output regardless of compilation target)

Reply:
I'm not sure. I think it depends on the web server and/or templating mechanisms that one use. The folks at https://gitter.im/scala-js/scala-js or https://stackoverflow.com/questions/tagged/scala.js might have more experience with that than I do.